src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionPublishButton.tag

```diff
@@ -0,0 +1,32 @@
+<%@ tag description="instructorFeedbacks and instructorHome - Feedback Session publish/unpublish button" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ tag import="teammates.common.util.Const" %>
+<%@ attribute name="publishButton" type="teammates.ui.template.FeedbackSessionPublishButton" required="true" %>
+<%@ attribute name="buttonType" required="true" %>
+<c:choose>
+    <c:when test="${publishButton.hasUnpublish}">
+        <a class="btn ${buttonType} btn-tm-actions session-unpublish-for-test"
+           href="${publishButton.unpublishLink}"
+           title="<%= Const.Tooltips.FEEDBACK_SESSION_UNPUBLISH %>"
+           data-toggle="tooltip"
+           data-placement="top"
+           onclick="return toggleUnpublishEvaluation(${publishButton.toggleUnpublishSessionParams});"
+           <c:if test="${not publishButton.allowedToUnpublish}">disabled="disabled"</c:if>>
+            Unpublish Results
+        </a>
+    </c:when>
+    <c:otherwise>
+        <div title="${publishButton.tooltipText}"
+             data-toggle="tooltip"
+             data-placement="top"
+             style="display: inline-block; padding-right: 5px;">
+            <a class="btn ${buttonType} btn-tm-actions session-publish-for-test<c:if test="${not publishButton.hasPublish}"> disabled</c:if>"
+               href="${publishButton.publishLink}"
+               <c:if test="${publishButton.hasPublish}">onclick="return togglePublishEvaluation(${publishButton.togglePublishSessionParams});"</c:if>
```

Comment:
I've also been thinking about these for awhile, where we pass values from Java into HTML -> JavaScript event handlers directly, this can be very hard to debug and can be very hard to maintainers to find where event handlers are being attached.

We might want to open an issue and make use of HTML5 data-* attributes and attach the event handlers via JavaScript directly. Should be better for maintainability. But again, not your fault or introduced by you.